projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -99,6 +99,18 @@ export class KeyNavigationGridController implements OnDestroy {
           this.removeActiveCell();
         });
 
+      fromEvent(this.grid, 'focusout')
+        .pipe(takeUntil(this.destroy$))
+        .subscribe(() => {
+          setTimeout(() => {
+            if (this.grid.contains(document.activeElement)) {
+              return;
+            }
+
+            this.removeActiveCell();
+          });
+        });
+
```

Comment:
I think this should work too and if we could leave a simple comment explaining what we are catching here would be good 
```suggestion
      fromEvent(this.grid, 'focusout')
        .pipe(debounceTime(0), takeUntil(this.destroy$))
        .subscribe(() => {
          if (this.grid.contains(document.activeElement)) {
            return;
          }

          this.removeActiveCell();
        });
```